### DISCUSSION TITLE
服务器错误：514（Frequency Capped）

### DISCUSSION THREAD

**Discussion by FuzzyPt233** [Coolapk-UWP/Coolapk-UWP#174] [Q&A]
![图片](https://user-images.githubusercontent.com/62923358/233285299-3b399ae3-ad29-4255-a32b-d1544d05db49.png)
我登录了客户端显示服务器错误，无法刷新？

**Comment by gxl946** [comment]
所以怎么解决，这软件还能不能用

**Comment by ghostvarg** [comment]
同样问题求救！
